content/data/friends-websites.yml

```diff
@@ -58,3 +58,7 @@
   about: >-
     เพราะนักพัฒนาไม่ได้สื่อสารแค่ภาษาเดียว Babel Coder จึงรวบรวมบทความและคอร์สอบรมอันหลากหลาย
     เพื่อตอบสนองการเขียนโปรแกรมในแต่ละภาษา พร้อมหลักการที่สำคัญเช่น คณิตศาสตร์และอัลกอริทึม
+
+- name: 🔥Firebase Thailand
```

Comment:
@jirawatee I am pretty sure if you have emoji at the beginning of the website's name, it is possible that it will be ordered to the last item in the list.
Would you be able to update the name to "Firebase Thailand 🔥" and  please make a force push to keep only one commit for this PR.
Then, I will merge your PR.

FYI, here is code that we sort all friends websites.
https://github.com/dotnetthailand/dotnetthailand.github.io/blob/main/src/components/MdxComponents/FriendsWebsites.tsx#L18

Reply:
Fixed https://github.com/dotnetthailand/dotnetthailand.github.io/pull/116/commits/f19712f26e9e1b38ae6f7b4b4223fc729c31711f